contrib/dbus-test-runner/patches/fix-test-output.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/tests/Makefile.am b/tests/Makefile.am
+index 4f4aad9..5e2d0b7 100644
+--- a/tests/Makefile.am
++++ b/tests/Makefile.am
+@@ -159,17 +159,17 @@ test-output: Makefile.am
+ 	@echo "#!/bin/sh -e" > $@
+ 	@echo "$(DBUS_RUNNER) --task cat --parameter \"$(top_srcdir)/src/dbus-test-runner.c\" --task-name \"cat1\" --task cat --parameter \"$(top_srcdir)/src/dbus-test-runner.c\" --task-name \"cat2\" > testcat.output.txt" >> $@
```

Comment:
the test-output test is flaking here, often cat2 does not manage to print out anything (i guess it races)

maybe best to just disable test-output from the suite